### DISCUSSION TITLE
Welcome to stdVBA Discussions!

### DISCUSSION THREAD

**Discussion by sancarn** [sancarn/stdVBA#30] [General]
Want to discuss feature requests? Implementations? Need help using stdVBA? This is a great place to do it :)

**Comment by rberke1216** [comment]
After 20 years of  coding VBA I have enjoyed its flexibility and been frustrated by its inadequacies.  

I have just read https://github.com/sancarn/stdVBA README.md
and I am blown away by your work. 

In the future I hope to make some small contributions to your project, but I am a github newbie and lack some knowledge that you probably take for granted.

For instance where can I download a dll that will support your stdArray idea?  

BTW I was surprised that your excellent ReadMe documentation did not contain a  "getting started" paragraph. Perhaps I just missed it because I am a GetHub newbie.

**Reply by sancarn** [reply]
Hi @rberke1216 There is no DLL. As many users are business users, I try to avoid dependencies where I can. If it were built into a DLL that'd be awesome though, and perhaps this is something that can be looked into with `twinBasic`...

I initially didn't include a getting started section as I thought importing CLS files into a VBA project was common knowledge, but you're not the only one who has fallen at this hurdle, so perhaps I was mistaken and need to include such a section!

## Getting started

If you download the project to disk, you can drag and drop the `.cls` files from the `src` folder to your own VBA project - this is the easiest way to get started.

Alternatively, you can open `testBuilder.xlsm` and build everything + run the tests from there.
